src/styles/index.scss

```diff
@@ -1,5 +1,6 @@
 body {
     margin: 0 auto;
+    background-color: #f5f5f5;
```

Comment:
please use variables

Reply:
![image](https://github.com/Real-Dev-Squad/website-status/assets/64726586/74d3c9c7-e2ba-4ec1-9c7c-78b35182a16f)
why are we resolving conversation without addressing them ?